core/src/main/resources/defaults/clustertemplates/aws/aws-opdb-700.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name": "CDP 1.0 - Operational Database template",
+  "description": "",
+  "type": "OPERATIONALDATABASE",
```

Comment:
This is an invalid value. What 'type' currently does, is it tells the UI, what icon it should present for this template. Right now there is no mapping for 'OPERATIONALDATABASE' and the enum 'ClusterTemplateV4Type' has no such value right now. Please change this to an existing value, like 'OTHER'.

Reply:
Ah ok. Thanks! I didn't realize it was an enum.